bears/general/CasingBear.py

```diff
@@ -0,0 +1,174 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.naming_conventions import (
+    to_camelcase, to_pascalcase, to_snakecase)
+from bears.general.AnnotationBear import AnnotationBear
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.results.SourceRange import SourceRange
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.results.HiddenResult import HiddenResult
+
+
+class CasingBear(LocalBear):
+
+    def run(self,
+            filename,
+            file,
+            dependency_results,
+            casing: str,
+            language: str,
+            coalang_dir: str=None):
+        """
+        Checks whether all identifier names (variables, classes, objects)
+        follow a certain naming convention.
```

Comment:
please make a note that this is experimental and doesn't distinguish between e.g. classes and variables. Also mention that people should report bugs to coala/coala-bears/issues/new , you can embed a preset title and description in the URL.